src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import java.util.Scanner;
+
+import java.io.FileNotFoundException;
+
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+import duke.parser.Parser;
+import duke.storage.Storage;
+
+import java.io.IOException;
```

Comment:
I think this should be placed with line 5, `import java.io.FileNotFoundException`.